DSPs/009-Internal-Audit.md

```diff
@@ -0,0 +1,28 @@
+# Internal Audit 
+
+**Author(s)**: @shresth31
+
+## Summary
+Create a knowledge base & checklist for dOrg's internal audits.
+
+## Why should we fund this?
+
+As dOrg is growing in number, with builders of varied levels of experties, and as the defi / smart-contract space is expanding with new and complex architectures, it's hard for all the builders to keep up with all new vulnerabilities in each project. 
+
+## Anything else you'd like to add?
+
+* The **Knowledge Base** will be structured like a blog. Every article would precisely describe a bug/ vulnerability with valid examples to repicate, ways to mitigate, and external resources to read more about it.
```

Comment:
Could this be the beginning of a dOrg blog? Like hosted under blog.dorg.tech?

Reply:
@benefacto I am not sure, as I would like to keep it isolated because of the following reasons:
*  It needs to be precise like a dictionary for smart contract vulnerabilities and not really a blog. 
*  I wouldn't like it to get mixed with other blogs by dOrg. 
*  Easier for open source contributions